### PR TITLE
hwxpxx: Initial implementation

### PR DESCRIPTION
**Pull request by jontrulson**
This module implements support for the Veris HWXPHTX Hardware Protocol
Humidity and Temperature Sensor family.  It uses MODBUS over an RS485
interface.

This module was developed using libmodbus 3.1.2, and the Veris HWXPHTX.

This sensor supports humidity, and optionally, temperature, slider
switch, and override switch reporting.  The HWXPHTX used to develop
this driver did not include the optional slider or override switches,
however support for them is provided.

Signed-off-by: Jon Trulson <jtrulson@ics.com>

### CODE REVIEW

File: src/hwxpxx/javaupm_hwxpxx.i [hunk 1]
```diff
@@ -0,0 +1,20 @@
+%module javaupm_t3311
```
Comment (Propanu) [inline src/hwxpxx/javaupm_hwxpxx.i]: Heh :smiley:

Comment (jontrulson) [inline src/hwxpxx/javaupm_hwxpxx.i]: Ooops.  Sorry, C/P error... as I don't actually write/test java :)

Comment (Propanu) [inline src/hwxpxx/javaupm_hwxpxx.i]: Done, no worries. Unfortunately Travis doesn't check modules based on 3rd party libs (yet).

**Comment by Propanu** [comment]
Merged, thanks!

**Pull request closed without merging**